code/game/objects/structures/window.dm

```diff
@@ -483,6 +483,15 @@
 	smooth = SMOOTH_TRUE
 	canSmoothWith = list(/obj/structure/window/fulltile, /obj/structure/window/reinforced/fulltile, /obj/structure/window/reinforced/tinted/fulltile)
 
+/obj/structure/window/reinforced/fulltile/ice
+	icon = 'icons/obj/smooth_structures/rice_window.dmi'
+	icon_state = "ice_window"
+	dir = 5
+	maxhealth = 150
+	fulltile = 1
+	smooth = SMOOTH_TRUE

```

Comment:
Apparently not, the other subtypes do this and not doing this causes it to not smooth.